plugins/kubernetes/app/models/kubernetes/template_filler.rb

```diff
@@ -171,10 +179,16 @@ def project
       @project ||= @doc.kubernetes_release.project
     end
 
+    def env
+      (container[:env] ||= [])
```

Comment:
Should we do the checking and replacing on all the containers, rather than just the first one?